src/activation_functions/activation_function.py

```diff
@@ -0,0 +1,10 @@
+from abc import ABC, abstractmethod
+
+class ActivationFunction(ABC):
+	@abstractmethod
+	def calculate(self, c):
+		pass
+
+	@abstractmethod
+	def calculateDerivative(self, x):
```

Comment:
Jakich konwencji nazewniczych używa się w Pythonie? Wczoraj pisząc UwZ trafiłem na taki artykuł:
https://medium.com/@dasagrivamanu/python-naming-conventions-the-10-points-you-should-know-149a9aa9f8c7

Jak zwykle pisze nasz Pythonowiec Rychu? :p

Reply:
Masz rację, powinno być `calculate_derivative`